gammapy/scripts/tests/test_make_bg_cube_models.py

```diff
@@ -0,0 +1,36 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+from astropy.tests.helper import pytest, remote_data
+from ...datasets import get_path
+from ..make_bg_cube_models import main as make_bg_cube_models_main
+from ...datasets import make_test_dataset
+
+try:
+    import scipy
+    HAS_SCIPY = True
+except ImportError:
+    HAS_SCIPY = False
+
+
+@pytest.mark.skipif('not HAS_SCIPY')
+@pytest.mark.parametrize("extra_options,something_to_test", [
```

Comment:
@cdeil I'm not sure about the second element in the tuple for parametrize. In fact, I'm not sure at all we should parametrize this test:

* On the one hand: it would be nice to test several options. Especially in the future, when more options become available.
* On the other hand, each test takes a few seconds to run, and we don't want the tests to take forever, right?

Suggestions?